software/speller/data_collection_platform/dcp/mp/shared.py

```diff
@@ -13,6 +14,7 @@
 character = Value("i", 0)
 frequency = Value("i", 0)
 phase = Value("i", 0)
```

Comment:
could you validate the "i" corresponds to floating type please, @TheFloatingString